doc/mediasdk-man.md

```diff
@@ -9423,6 +9423,8 @@ The `PlatformCodeName` enumerator itemizes Intel® microarchitecture code names.
 `MFX_PLATFORM_JASPERLAKE`   | Jasper Lake
 `MFX_PLATFORM_ELKHARTLAKE`  | Elkhart Lake
 `MFX_PLATFORM_TIGERLAKE`    | Tiger Lake
+`MFX_PLATFORM_ROCKETLAKE`   | Rocket Lake
```

Comment:
Usually such lines guarded by MFX_VERSION >= MFX_VERSION_NEXT macro check. But for this particular change, I believe addition is relatively small. So I believe it is ok.